Lib/ldap/controls/libldap.py

```diff
@@ -26,14 +28,18 @@ class AssertionControl(RequestControl):
   """
 
   controlType = ldap.CONTROL_ASSERT
-  def __init__(self,criticality=True,filterstr='(objectClass=*)'):
+
+  def __init__(
+    self, criticality: bool = True, filterstr: str = '(objectClass=*)'
+  ) -> None:
     self.criticality = criticality
     self.filterstr = filterstr
 
-  def encodeControlValue(self):
-    return _ldap.encode_assertion_control(self.filterstr)
+  def encodeControlValue(self) -> bytes:
+    return _ldap.encode_assertion_control(self.filterstr)  # type: ignore
 
-KNOWN_RESPONSE_CONTROLS[ldap.CONTROL_ASSERT] = AssertionControl
+# FIXME: This is a request control though?
+#KNOWN_RESPONSE_CONTROLS[ldap.CONTROL_ASSERT] = AssertionControl
```

Comment:
FIXME should be fixed. Just pointing it out, not sure what to do here.

Reply:
Yeah, I'm not sure that I'm right here, so I kind of left it as a marker for the maintainers to have a look. I guess the "fix" would be just deleting the lines, but not sure